pytripgui/config_vc/config_cont.py

```diff
@@ -1,10 +1,11 @@
+from copy import deepcopy
 import logging
 logger = logging.getLogger(__name__)
 
 
 class ConfigController(object):
     def __init__(self, model, view):
-        self.model = model
+        self.model = deepcopy(model)
```

Comment:
Why do you need deepcopy here ?

Reply:
Because while u editing trip_configs, you operate on a copy of configs.
When you click "Save" button, this copy will bu used as config,
when you click "Cancel" button, then this copy will be thrown out.

Reply:
add a comment explaining that